### PR TITLE
Added CLI file and script to setup.cfg.

### PR DESCRIPTION
**Pull request by cthev456**
## Description
The main goal of this task was to build a command line tool for the Wearipedia project in order to streamline user experience in a simple interface. Thus, it eliminates the need to learn how to code, improving accessibility all around. The target of the CLI is to be able to run “pip install wearipedia” to install dependencies and then the newly installed CLI would be able to interpret user commands that begin with “wearipedia.” When building this tool in Python, I utilized the Python argparse library as the main framework for the CLI since it preserves industry standards in a CLI. I also needed to add scripts and entry points inside setup.cfg to trigger my parsing program. Finally, I needed to account for certain use cases which are of course, subject to change as more complexity is added to the Wearipedia project. In terms of coding these portions, it was straightforward to address what kinds of data a user wants, how they insert their login details, specifying time range parameters, etc using argparse. 

The most difficult part of this task was figuring out how to set up the script to auto-run my CLI changes inside the codebase. There was a lot of conflicting info online about scripting versus entry points for Python CLIs since there are many different ways to achieve the same goal. It became even more difficult when I discovered we were using Poetry to package everything which confused me further about how I should add changes to the already setup config files. However, thanks to Rodrigo, I was able to add scripts and entry points to setup.cfg which does not interfere with the Poetry setup inside of pyproject.toml.

## Related Issue

N/A

### CODE REVIEW

File: wearipedia/cl_parser.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+    # check if output file exists and is valid
+    print_to_console = True
+    if arg_dict["output_path"]:
+        output_file = arg_dict["output_path"]
+        if has_valid_extension(output_file):
+            try:
+                # Get the data based on what type of data is requested and if it is synthetic/real
+                with open(output_file, "w+") as f:
+                    f.write(data)
```
[the remark this answers is not in the document]
Comment (cthev456) [reply wearipedia/cl_parser.py:134]: I pushed my changes to my branch yesterday, do I need to create another pull request in order for them to be seen here?

[the remark this answers is not in the document]
Comment (rodrigo-castellon) [reply wearipedia/cl_parser.py:134]: I don't see them popping up. Does that commit appear on your end in this pull request?

[the remark this answers is not in the document]
Comment (cthev456) [reply wearipedia/cl_parser.py:134]: Ok, something weird that I'll have to show you later but it seems my forked repository is no longer linked to this main one which is why my changes are not showing. I cannot pull from the main nor do my commits show here but my forked version it does.

[the remark this answers is not in the document]
Comment (rodrigo-castellon) [reply wearipedia/cl_parser.py:134]: Ohhh I know what happened. The repo went public. You can just make a new PR with your updated changes.

**Pull request closed without merging**